src/applications/facility-locator/components/VABenefitsCall.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { parsePhoneNumber } from '../utils/phoneNumbers';
+import Telephone from '@department-of-veterans-affairs/formation-react/Telephone';
+
+const { formattedPhoneNumber, extension, contact } = parsePhoneNumber(
+  '800-827-1000',
+);
+
+export default function VABenefitsCall() {
+  return (
+    <div>
+      <p className="p1">
+        <strong>To check on the status of your claim or appeal</strong>, use our{' '}
+        <a href="/claim-or-appeal-status">online claim status tool</a>.
+      </p>
+      <p className="p1">
+        <strong>To get help with benefits</strong>, call{' '}
```

Comment:
Would it be better to use `&nbsp;` instead of `{' '}`?

Reply:
Well, `{' '}` is added  by esli
<img width="1384" alt="Screen Shot 2020-07-15 at 2 32 17 PM" src="https://user-images.githubusercontent.com/100468/87587710-135eb280-c6a8-11ea-9465-a95675e4a528.png">
nt and is widely used in other components